pub_keys/marisa.pub

```diff
@@ -1 +1 @@
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCReyc0uxOCcjI3fdaBim4AIvxe0Xsw65tvATO3oF0ztTu/zSTZrEHIRkhcBTvLbFhDLC6VUI22Z7r2BndzcEWBFw6yLIbBzYpaDULl8Vfh4jUF/y+Vt0MnDGfS6kspyCnNExzfsmPZuswTFonqeN64/YjSmZubVTfa6/Mwzsh0rlj1JXWRY/UQ83D5j9Bq/wB3QmspGQ2d6atFhOoDtYIPL8OhXlkwHKamkQP20rViTI9/8b29Y8ur0rCnwvSO4U9zmwZANeKMMx+HNQXK6Txqhl6Ybfr66CMGfdrJFqTzubpZmnsixrAx+Bzo3qfVY5spit9MpHuBW124SqsgH8pL marisa@ThinkPad-E480
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCReyc0uxOCcjI3fdaBim4AIvxe0Xsw65tvATO3oF0ztTu/zSTZrEHIRkhcBTvLbFhDLC6VUI22Z7r2BndzcEWBFw6yLIbBzYpaDULl8Vfh4jUF/y+Vt0MnDGfS6kspyCnNExzfsmPZuswTFonqeN64/YjSmZubVTfa6/Mwzsh0rlj1JXWRY/UQ83D5j9Bq/wB3QmspGQ2d6atFhOoDtYIPL8OhXlkwHKamkQP20rViTI9/8b29Y8ur0rCnwvSO4U9zmwZANeKMMx+HNQXK6Txqhl6Ybfr66CMGfdrJFqTzubpZmnsixrAx+Bzo3qfVY5spit9MpHuBW124SqsgH8pL marisa
```

Comment:
why did you need this? The public key seems good as it is.

Reply:
I thought it would be more homogeneous with the other keys this way and that the username would not affect to the key itself. We can don't do it.

What risks are you thinking on?

Reply:
No idea if there's any risk or not. If your key had that email in there, I guess it's fine. However, if you choose to remove that part, I'd remove it all, including your name.

@enricostano you might know a bit more about pub key contents than me.

Reply:
In any case, we can move out of this PR this change and do it in a separate PR, no @mperezv?

Reply:
I'm OK with the change, probably better if done in another PR.

The only thing that's important to me is that at least the name `marisa` is left there. It's quite useful while auditing servers. It should be our standard.

Reply:
agree with @enricostano 's suggestion. Would you mind adding the SSH key in a separate PR before we merge? better to have a purpose-specific PR for each thing.

Reply:
On it! Give me a few minutes to remember how to cherry-pick and I'll do de PR :smiley: